client/my-sites/upgrades/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -167,21 +189,22 @@ class EditContactInfoFormCard extends React.Component {
 					isPrimary: true
 				}
 			],
-			oldEmail = this.props.contactInformation.email,
-			newEmail = formState.getFieldValue( this.state.form, 'email' );
+			currentEmails = this.getCurrentEmails();
 
 		let text;
-		if ( oldEmail === newEmail ) {
+		if ( this.hasEmailChanged() ) {
 			text = translate( 'To finish this process, this change will need to be confirmed by an email ' +
 				'sent to {{strong}}%(email)s{{/strong}}. Please make sure you have access to it.', {
-					args: { email: newEmail }, components: { strong }
+					args: { email: currentEmails }, components: { strong }
 				}
 			);
 		} else {
+			const newEmail = formState.getFieldValue( this.state.form, 'email' );
+
 			text = translate( 'To finish this process, this change will need to be confirmed by emails ' +
```

Comment:
@yoavf That was partly my fault, party GitHub's way of showing this change. First of all - IIRC, the string _did_ change, just on a line below - that was the continuation of this string. So, @a8ci18n was right, it's just from the small context that GitHub showed it, it seemed like a bug. So, sorry for the false alarm 🙇